python/tank/platform/constants.py

```diff
@@ -135,3 +135,6 @@
 
 # hook that is executed whenever a cache location should be determined
 CACHE_LOCATION_HOOK_NAME = "cache_location"
+
+# environment variable that if set, enables debug logging in the engine
+ENGINE_DEBUG_LOGGING_ENV_VAR = "TK_SHOW_ENGINE_DEBUG"
```

Comment:
This feels like a pretty complicated name.  Would something like TK_DEBUG and having it turn on debug across the board (root logger as well) make sense?